build.psm1

```diff
@@ -1012,7 +1012,7 @@ function Start-PSPester {
     # All concatenated commands/arguments are suffixed with the delimiter (space)
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-linux`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=.Xml.XDocument.dll; SideIndicator=<=}.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 121
121:         $diffs | Should -BeExactly $null
```